test/Microsoft.AspNetCore.Sockets.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -424,6 +426,171 @@ public class HttpConnectionDispatcherTests
             Assert.Equal(MessageType.Close, messages[3].Type);
         }
 
+        [Fact]
+        public async Task UnauthorizedConnectionFailsToStartEndPoint()
```

Comment:
Its either `/sse` `/poll` or `/ws` these tests don't test if a transport can send, they test if you can do something with the endpoint after authorization, so we don't care what path as long as it works.